### PR TITLE
Feature/gal 1334

### PR DESCRIPTION
**Pull request by Aymene-n**
## Type of changes

- [ ] Refactoring (non-breaking change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)

## Checklist

- [ ] I am familiar with the [contributing guidelines](../docs/CONTRIBUTING.md)
- [ ] I have followed the code style of the project
- [ ] I have added tests to cover my changes
- [ ] I have updated the documentation accordingly
- [ ] This PR is a result of pair or mob programming

---

## Sensitive Information Declaration

To ensure the utmost confidentiality and protect your and others privacy, we kindly ask you to NOT including [PII (Personal Identifiable Information) / PID (Personal Identifiable Data)](https://digital.nhs.uk/data-and-information/keeping-data-safe-and-benefitting-the-public) or any other sensitive data in this PR (Pull Request) and the codebase changes. We will remove any PR that do contain any sensitive information. We really appreciate your cooperation in this matter.

- [ ] I confirm that neither PII/PID nor sensitive data are included in this PR and the codebase changes.

### CODE REVIEW

**Review by Amar-NHS — APPROVED** [review]

File: terraform/src/cancerSignalOriginAddLambda/cancerSignalOriginAddLambda.js [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+          Grail_Code: { S: item.Grail_Code || "" },
+          Grail_Heading: { S: item.Grail_Heading || "" },
+          Grail_Subheading: { S: item.Grail_Subheading || "" },
+          Cso_Result_Snomed_Code_And_Preferred_Term: {
+            S: item.Cso_Result_Snomed_Code_And_Preferred_Term || "",
+          },
+          Cso_Result_Friendly: { S: item.Cso_Result_Friendly || "" },
+          Created_By: { S: item.Created_By || "" },
+          Start_Date: { S: "Date Now" },
+          End_Date: { S: "End date" },
```
Comment (chenglawson1) [inline terraform/src/cancerSignalOriginAddLambda/cancerSignalOriginAddLambda.js:95]: Why are start and end dates hardcoded strings that are not iso datetime as specified in the Confluence page?
Start_Date should be the current datetime in iso format.
End_Date can be set to "" for now.

File: terraform/src/cancerSignalOriginAddLambda/cancerSignalOriginAddLambda.js
Comment (chenglawson1) [inline terraform/src/cancerSignalOriginAddLambda/cancerSignalOriginAddLambda.js:1]: No unit tests?

**Review by chenglawson1 — COMMENTED** [review]
No unit tests?

**Comment by chenglawson1** [comment]
> No unit tests?

Unit tests added now, thanks.

**Review by Amar-NHS — APPROVED** [review]

**Pull request merged**